components/features/Dashboard/Window/Resizers/Resizers.module.css

```diff
@@ -0,0 +1,76 @@
+.resizer {
+  position: absolute;
```

Comment:
The reason the resizer has to be absolutely positioned, is because we need to place it in such a way that it is one the edges of the parent container.

Reply:
Oh alright rashad, thanks for clearing the air on that. 😁